huaweicloud/services/acceptance/modelarts/resource_huaweicloud_modelarts_service_test.go

```diff
@@ -0,0 +1,212 @@
+					resource.TestCheckResourceAttr(rName, "config.0.envs.a", "1"),
+					resource.TestCheckResourceAttr(rName, "config.0.envs.b", "2"),
+					resource.TestCheckResourceAttrPair(rName, "additional_properties.0.smn_notification.0.topic_urn",
+						"huaweicloud_smn_topic.test", "id"),
+					resource.TestCheckResourceAttr(rName, "additional_properties.0.smn_notification.0.events.0", "3"),
+					resource.TestCheckResourceAttr(rName, "additional_properties.0.log_report_channels.0.type", "LTS"),
+					resource.TestCheckResourceAttr(rName, "schedule.0.type", "stop"),
+					resource.TestCheckResourceAttr(rName, "schedule.0.duration", "1"),
+					resource.TestCheckResourceAttr(rName, "schedule.0.time_unit", "HOURS"),
+					resource.TestCheckResourceAttrSet(rName, "owner"),
+					resource.TestCheckResourceAttrSet(rName, "access_address"),
+					resource.TestCheckResourceAttrSet(rName, "invocation_times"),
+					resource.TestCheckResourceAttrSet(rName, "failed_times"),
+					resource.TestCheckResourceAttrSet(rName, "is_shared"),
+					resource.TestCheckResourceAttrSet(rName, "shared_count"),
+					resource.TestCheckResourceAttrSet(rName, "is_free"),
+					resource.TestCheckResourceAttrSet(rName, "publish_at"),
+					resource.TestCheckResourceAttrSet(rName, "due_time"),
+					resource.TestCheckResourceAttrSet(rName, "transition_at"),
+				),
+			},
+			{
+				Config: testModelartsService_basic_update(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttr(rName, "description", "This is a acc test updated"),
+					resource.TestCheckResourceAttr(rName, "status", "stopped"),
+					resource.TestCheckResourceAttr(rName, "config.0.envs.b", "3"),
+					resource.TestCheckResourceAttr(rName, "additional_properties.0.smn_notification.0.events.0", "3"),
+					resource.TestCheckResourceAttr(rName, "additional_properties.0.smn_notification.0.events.1", "7"),
+					resource.TestCheckResourceAttr(rName, "schedule.0.type", "stop"),
+					resource.TestCheckResourceAttr(rName, "schedule.0.duration", "2"),
+					resource.TestCheckResourceAttr(rName, "schedule.0.time_unit", "HOURS"),
+				),
+			},
+			{
+				ResourceName:            rName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"change_status_to"},
+			},
+		},
+	})
+}
+
+func testModelartsService_basic(name string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_smn_topic" "test" {
+  name         = "%s"
+  display_name = "This is a acc test"
+}
+
+resource "huaweicloud_modelarts_service" "test" {
+  name        = "%s"
+  description = "This is a acc test"
+  infer_type  = "real-time"
+
+  config {
+    specification  = "modelarts.vm.gpu.p4u8.container"
+    instance_count = 1
+    weight         = 100
+    model_id       = "48a44dd4-b15d-48b7-989a-f5b5f2d9a659"
```

Comment:
please replace the UUID after #3171 is merged.

Reply:
ok